test/adapters/stochastic.spec.js

```diff
@@ -25,14 +25,16 @@ describe('stochastic adapter options', function() {
         });
     });
 
-    it('supports starting at :beginning:', function() {
+    it('complains about starting at :beginning:', function() {
         return check_juttle({
             program: 'read stochastic -from :beginning: '+
                      '-to :1970-01-01T00:00:10.000Z: ' +
                      '| reduce count() | view text'
+        }).then(function(res) {
```

Comment:
There's this thing called `test_utils.expect_to_fail` that can simplify tests like this if you want.